xbmc/interfaces/legacy/ListItem.cpp

```diff
@@ -482,6 +482,8 @@ namespace XBMCAddon
             videotag.SetTrailer(value);
           else if (key == "path")
             videotag.SetPath(value);
+          else if (key == "filenameandpath")
```

Comment:
Btw how does it differ from "path"? https://github.com/faush01/xbmc/blob/3c82c4fbfdff48a4190a0e7cda2bdde20f338911/xbmc/interfaces/legacy/ListItem.h#L634
I would think it holds the same info? (Which isn't your fault, just checking for docs sake)